docs/english/option/index.md

```diff
@@ -0,0 +1,20 @@
+# Options
+## Use ActivtyPub Relay as a limited Hashtag Relay
+After you deploy this option, ActivtyPub Relay will rebroadcast posts that include allowed hashtags only.
+
+### How to deploy
+Move to the cloned directory of your computer, and set ALLOWED_HASHTAGS environment variable as below.
+
+```
+ALLOWED_HASHTAGS=#activitypub,#activitypub_relay
+```
+
+- `#` is needed for each tag
+- Separate each tag with commas
+- Do **not** make space after commas
```

Comment:
Maybe chage to `Do **not** include space before or after commas.`  that more better?

Reply:
That would be better !